adapt_med_seg/models/lightning_model.py

```diff
@@ -32,7 +32,7 @@ def load_from_checkpoint(checkpoint_path: str, model_name: str, modalities: list
         model = SegVolLightning(model_name, modalities, tasks, test_mode, **kwargs)
 
         checkpoint = torch.load(checkpoint_path, map_location=model._device)
-        print(checkpoint["state_dict"].keys())
+        # print(checkpoint["state_dict"].keys())
```

Comment:
not related but I think it was left inside accidentally. Prints ~50 lines of unreadable module paths